packages/ui/components/dao/DaoCard.tsx

```diff
@@ -1,5 +1,129 @@
-export interface DaoCardProps {}
+import clsx from 'clsx'
+import Link from 'next/link'
+import { useTranslation } from 'react-i18next'
 
-export const DaoCard = ({}: DaoCardProps) => {
-  return <p>placeholder</p>
+import {
+  DaoCardMemberCheck,
+  DaoCardPin,
+  DaoCardProposals,
+  Governance,
+  SubDaoArrow,
+} from '@dao-dao/icons'
+
+export interface SubDaoInfo {
+  parentDaoHref: string
+  parentDaoImageUrl: string
+}
+
+export interface DaoCardProps {
+  name: string
+  description: string
+  imageUrl: string
+  established: Date
+  href: string
+
+  junoBalance: number
+  proposalCount: number
+
+  subDaoInfo?: SubDaoInfo
+
+  pinned: boolean
+  onPin: () => null
+}
+
+export const DaoCard = ({
+  name,
+  description,
+  imageUrl,
+  href,
+  established,
+  junoBalance,
+  proposalCount,
+  pinned,
+  onPin,
+  subDaoInfo,
+}: DaoCardProps) => {
+  const { t } = useTranslation()
+
+  return (
+    <Link href={href} passHref>
+      <a className="flex relative flex-col justify-between items-center py-7 px-6 w-[260px] h-[328px] bg-background-secondary hover:bg-background-interactive-hover active:bg-background-interactive-pressed rounded-md outline-transparent hover:outline-border-interactive-hover active:outline-border-interactive-focus outline transition-all">
+        <div
+          className={clsx(
+            'flex absolute top-0 left-0 flex-row items-center p-3 w-full',
+            {
+              'justify-between': !!subDaoInfo,
+              'justify-end': !subDaoInfo, // Keep the pin and member check at the end if no subDao info.
+            }
+          )}
+        >
+          {subDaoInfo && (
+            <Link href={subDaoInfo.parentDaoHref} passHref>
+              <a title={t('info.gotoParent')}>
+                <SubDaoArrow className="fill-icon-interactive-disabled" />
+              </a>
+            </Link>
+          )}
+          <div className="flex flex-row gap-3">
+            <button
+              onClick={onPin}
+              title={pinned ? t('info.pinned') : t('info.pin')}
+            >
+              <DaoCardPin
+                className={clsx('w-4 h-4', {
+                  'fill-icon-secondary': !pinned,
+                  'fill-icon-interactive-active': pinned,
```

Comment:
```suggestion
                  'text-icon-secondary': !pinned,
                  'text-icon-interactive-active': pinned,
```

Rewrite icon to use `currentColor` internally and change this to use text color classes.